papis/commands/update.py

```diff
@@ -86,100 +369,163 @@ def run(document: papis.document.Document,
 @papis.cli.doc_folder_option()
 @papis.cli.all_option()
 @papis.cli.sort_option()
-@papis.cli.bool_flag("--auto",
-                     help="Try to parse information from different sources")
-@click.option("--from", "from_importer",
-              help="Add document from a specific importer ({})".format(
-                  ", ".join(papis.importer.available_importers())
-              ),
-              type=(click.Choice(papis.importer.available_importers()), str),
-              nargs=2,
-              multiple=True,
-              default=(),)
-@click.option("-s", "--set", "set_tuples",
-              help="Update document's information with key value. "
-                   "The value can be a papis format.",
-              multiple=True,
-              type=(str, str),)
-@papis.cli.bool_flag("-b", "--batch",
-                     help="Batch mode, do not prompt or otherwise")
-def cli(query: str,
-        git: bool,
-        doc_folder: Tuple[str, ...],
-        from_importer: List[Tuple[str, str]],
-        batch: bool,
-        auto: bool,
-        _all: bool,
-        sort_field: Optional[str],
-        sort_reverse: bool,
-        set_tuples: List[Tuple[str, str]],) -> None:
-    """Update a document from a given library."""
-
-    documents = papis.cli.handle_doc_folder_query_all_sort(query,
-                                                           doc_folder,
-                                                           sort_field,
-                                                           sort_reverse,
-                                                           _all)
+@papis.cli.bool_flag("--auto", help="Try to parse information from different sources")
+@click.option(
+    "--from",
+    "from_importer",
+    help="Add document from a specific importer ({})".format(
+        ", ".join(papis.importer.available_importers())
+    ),
+    type=(click.Choice(papis.importer.available_importers()), str),
+    nargs=2,
+    multiple=True,
+    default=(),
+)
+@click.option(
+    "-s",
+    "--set",
+    "to_set",
+    help="Set a key's value. ",
+    multiple=True,
+    type=(str, str),
+)
+@click.option(
+    "-d",
+    "--drop",
+    "to_drop",
+    help="Drop a key-value pair.",
+    multiple=True,
+    type=str,
+)
+@click.option(
+    "-p",
+    "--append",
+    "to_append",
+    help="Append an item to a string or a list. ",
+    multiple=True,
+    type=(str, str),
+)
+@click.option(
+    "-r",
+    "--remove",
+    "to_remove",
+    help="Remove an item from a list.",
+    multiple=True,
+    type=(str, str),
+)
+@click.option(
+    "-n",
+    "--rename",
+    "to_rename",
+    help="Rename a list item.",
```

Comment:
```suggestion
    help="Replace a value in the document key.",
```
? Maybe we should rename this to `--replace`? Sounds a bit more general in case we ever want to do other things here..

Reply:
Hm, that's a good point, we can leave it as is.